React/tic-tac/src/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './scss/index.scss';
+
+import Header from '@components/header'; // eslint-disable-line import/first
+import Game from '@screens/game/';
+
+ReactDOM.render(
+    <React.Fragment>
```

Comment:
Try to return only one React component here to keep the index file as clean as possible. You can move these elements to an `App` component that handles the whole app layout.